contracts/interfaces/IPendingOwnable.sol

```diff
@@ -21,3 +21,23 @@ interface IPendingOwnable {
 
     function renounceOwnership() external;
 }
+
+interface IPendingOwnableUpgradeable {
+    event PendingOwnerSet(address indexed pendingOwner);
+    event OwnershipTransferred(
+        address indexed previousOwner,
+        address indexed newOwner
+    );
+
+    function owner() external view returns (address);
+
+    function pendingOwner() external view returns (address);
+
+    function setPendingOwner(address pendingOwner) external;
+
+    function revokePendingOwner() external;
+
+    function becomeOwner() external;
+
+    function renounceOwnership() external;
+}
```

Comment:
Same, please put this in a separate file, because when you verify the contract it will increase the length of the contract for no reason